addons/sale_stock/views/sale_order_views.xml

```diff
@@ -27,16 +27,16 @@
                 <xpath expr="//field[@name='commitment_date']" position="after">
                     <field name="effective_date" groups="sale.group_sale_order_dates"/>
                 </xpath>
-                <xpath expr="//page/field[@name='order_line']/form/group/group/field[@name='tax_id']" position="before">
+                <xpath expr="//page/field[@name='order_line']/form/group/group/label[@for='customer_lead']" position="before">
```

Comment:
Why not use `field[@name='customer_lead']` ?

Reply:
> Why not use `field[@name='customer_lead']` ?

because this field is going to be inherited separately in form view as well as in tree view of one2many. 
Thanks

Reply:
@mba-odoo I mean replace

`//page/field[@name='order_line']/form/group/group/label[@for='customer_lead']`

with

`//page/field[@name='order_line']/form/group/group/field[@name='customer_lead']`

Reply:
@awa-odoo , No because we want `product_packaging `  before the label, not before the field. 
on main view structure is like: `<label for="customer_lead"/><div><field name="customer_lead" class="oe_inline"/> days</div>`
Thanks